nidirect_common/src/InvalidateTaxonomyListCacheTags.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Drupal\nidirect_common;
+
+use Drupal\Core\Cache\Cache;
+
+/**
+ * Class InvalidateTaxonomyListCacheTags.
+ */
+class InvalidateTaxonomyListCacheTags {
```

Comment:
Now this class has moved to a service, you should be able to inject the cache_tags.invalidator service as an argument in the services.yml file and wire it up to an internal class variable for use in any functions through the usual constructor + static create functions.